### PR TITLE
Changed TargetType to match /etc/os-releases, single target for AliyunLinux 2 and 3

### PR DESCRIPTION
**Pull request by dark-vex**
**What type of PR is this?**

/kind cleanup

/kind feature

**Any specific area of the project related to this PR?**

/area build

**What this PR does / why we need it**:

Implementing the changes suggested on a previously merged PR: https://github.com/falcosecurity/driverkit/pull/262#discussion_r1148889374

### CODE REVIEW

**Comment by dark-vex** [comment]
Yes absolutely, I will do that

**Comment by dark-vex** [comment]
Don't get scared about the changes to `Example_configs.md` file :D I have organized it in alphabetical order and I have added also Alma and Rocky Linux which were missing as well

File: Example_configs.md [hunk 1]
```diff
@@ -1,135 +1,134 @@
...
-driverversion: master
-```
+Example configuration file to build both the Kernel module and eBPF probe for Archlinux.
+Note: archlinux target uses the [Arch Linux Archive](https://wiki.archlinux.org/title/Arch_Linux_Archive) to fetch
+all ever supported kernel releases.
+For arm64, it uses an user-provided mirror, as no official mirror is available: http://tardis.tiny-vps.com/aarm/.
+The mirror has been up and updated since 2015.
 
-## oracle linux 8
+NOTE: ArchLinux has updated to Linux Kernel 6.0+, which appears to enforce generation of BTF for kmod too. The typical Debian container builders in the repo currently do not work for building ArchLinux drivers as a result. While an agreement is being reached on how to handle this upstream, using a custom `--builderimage` with ArchLinux drivers works for now. For example:
```
[the remark this answers is not in the document]
Comment (FedeDP) [reply Example_configs.md:76]: (Last change and we are good to go! Thanks for this btw :) )

[the remark this answers is not in the document]
Comment (dark-vex) [reply Example_configs.md]: should be fine now :)

**Review by FedeDP — APPROVED** [review]
/approve

Thank you!

**Pull request merged**